academy/3-my-own-chain/migration.md

```diff
@@ -1261,13 +1274,17 @@ func PerformMigration(ctx sdk.Context, k keeper.Keeper, storedGameChunk uint64,
 }
 ```
 
-Note that it does not panic in case of an error. To avoid carrying on a faulty state, the caller of this function will have to handle the panic.
+<HighlightBox type="note">
+
+This does not panic in case of an error. To avoid carrying on a faulty state, the caller of this function will have to handle the panic.
+
+</HighlightBox>
 
 You have in place the functions that will handle the store migration. Now you have to set up the chain of command for these functions to be called by the node at the right point in time.
 
 ### Consensus version and name
 
-The `upgrade` module keeps in its own store the [different module versions](https://docs.cosmos.network/master/core/upgrade.html#tracking-module-versions) that are currently running. To signal an upgrade, your module needs to return a different value when queried by the `upgrade` module. Change it from `2` to `3`, or whichever number works for you. First keep both these values in their respective locations:
+The `upgrade` module keeps in its own store of the [different module versions](https://docs.cosmos.network/master/core/upgrade.html#tracking-module-versions) that are currently running. To signal an upgrade, your module needs to return a different value when queried by the `upgrade` module. Change it from `2` to `3`, or whichever number works for you. First keep both these values in their respective locations:
```

Comment:
No no.
It is:
The `upgrade` module keeps the different module versions. Where? In its own store.

Reply:
Corrected.